package.json

```diff
@@ -22,5 +22,8 @@
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-react-refresh": "^0.4.3",
     "prettier": "^3.0.3"
+  },
+  "dependencies": {
```

Comment:
This should only be installed on the frontend, and not on the root package/package-lock file. Could you run `npm uninstall @ant-design/icons` within the root directory? That *should* also resolve the conflicts between this branch and main